plugins/replay/src/main/java/forklift/replay/ReplayLogBuilder.java

```diff
@@ -65,8 +66,13 @@ public ReplayLogBuilder(ForkliftMessage msg, Consumer consumer, List<String> err
         // carry a description of the original source across restarts
         props.putIfAbsent("source-description", sourceDescription);
 
+
+        if (!props.containsKey("forklift-replay-step-count")) { // new messages are version 3
+            props.putIfAbsent("forklift-replay-version", "3");
+        }
```

Comment:
Do you need to put forklift-replay-version of 2 here if you there is a step count present but no version?